src/app/app.component.ts

```diff
@@ -4,7 +4,7 @@ import { Component } from '@angular/core';
 @Component({
   selector: 'app-root',
   templateUrl: './app.component.html',
-  styleUrls: ['./app.component.scss']
+  styleUrls: ['./app.component.scss',]
```

Comment:
Maybe a typo? Or there might be something about typescript/javascript I don't know yet and this does something :-)